core/kazoo_services/src/kz_services_quantities.erl

```diff
@@ -510,7 +510,7 @@ calculate_faxbox_updates(JObj, Updates) ->
     case kz_doc:type(JObj) =:= <<"faxbox">> of
         'false' -> Updates;
         'true' ->
-            Key = [<<"faxes">>, <<"faxbox">>],
+            Key = [<<"faxes">>, <<"mailbox">>],
```

Comment:
If you want, these functions appear to be building a `kz_json:flat_proplist()`, essentially `[{path(), json_term()}]`. Could be nice to have a more specific return type used.

Also, unrelated, but I notice `calculate_qubicle_*_updates/2`; I wonder, could we leverage AMQP to do these calculations and let the apps that want to be quantifiable bind in and be called to calculate things? Then it becomes a `kz_amqp_worker:collect` with a binding key like `services.calculate.{PVT_TYPE}` and the payload is the JObj. Then each app that wants to be "billable" has a listener for the pvt_type's they are interested in, also allowing custom apps to add billing for their doc types.